src/controllers/chat.js

```diff
@@ -1,5 +1,5 @@
 exports.get = (req, res) => {
```

Comment:
Is the chat part of this issue ??

Reply:
No

Reply:
so you should remove it from this pull request

Reply:
you should remove any file that is not part of this issue like the chat files
and keep every pull request simple and related to one issue